### ISSUE TITLE
static files cannot be shown after installing

### ISSUE THREAD

**Issue by skul169** [orangehrm/orangehrm#266]
After installing, static files (image, css, js....) cannot be shown because look url is wrong.
http://my-domain.com/var/www/orangehrm/symfony/web/themes/default/images/login/logo.png

How i can fix it ? Thanks.
here is virtualhost:
```
<VirtualHost *:80>
  ServerName my-domain.com
  DocumentRoot "/var/www/orangehrm/symfony/web"
  DirectoryIndex index.php
  <Directory "/var/www/orangehrm/symfony/web">
    Options -Indexes +FollowSymLinks +MultiViews
    Require all granted
    AllowOverride All
  </Directory>

  Alias /sf "/var/www/orangehrm/symfony/lib/vendor/symfony/data/web/sf"
  <Directory "/var/www/orangehrm/symfony/lib/vendor/symfony/data/web/sf">
    AllowOverride All
    Require all granted
  </Directory>
</VirtualHost>
```

and i got this error in log file also
```
[30-Jul-2018 00:05:55 Asia/Singapore] Action "var/www" does not exist.
[30-Jul-2018 00:05:55 Asia/Singapore] PHP Fatal error:  Uncaught exception 'sfStopException' in /var/www/orangehrm/symfony/lib/vendor/symfony/lib/filter/sfBasicSecurityFilter.class.php:96
Stack trace:
#0 /var/www/orangehrm/symfony/lib/vendor/symfony/lib/filter/sfBasicSecurityFilter.class.php(55): sfBasicSecurityFilter->forwardToLoginAction()
#1 /var/www/orangehrm/symfony/cache/orangehrm/prod/config/config_core_compile.yml.php(1044): sfBasicSecurityFilter->execute(Object(sfFilterChain))
#2 /var/www/orangehrm/symfony/cache/orangehrm/prod/config/config_core_compile.yml.php(1009): sfFilterChain->execute()
#3 /var/www/orangehrm/symfony/cache/orangehrm/prod/config/config_core_compile.yml.php(1044): sfRenderingFilter->execute(Object(sfFilterChain))
#4 /var/www/orangehrm/symfony/cache/orangehrm/prod/config/config_core_compile.yml.php(685): sfFilterChain->execute()
#5 /var/www/orangehrm/symfony/lib/vendor/symfony/lib/exception/sfError404Exception.class.php(49): sfController->forward('default', 'error404')
#6 /var/www/orangehrm/symfony/cache/orangehrm/prod/config/config_cor in /var/www/orangehrm/symfony/lib/vendor/symfony/lib/filter/sfBasicSecurityFilter.class.php on line 96
```

**Comment by samanthajayasinghe** [comment]
@skul169 
Why you create the alias 'sf' ? As I understand it cause the issue I mean following vhost config 
Alias /sf "/var/www/orangehrm/symfony/lib/vendor/symfony/data/web/sf"

**Comment by RajithaKumara** [comment]
Related to #584 
Fixed  #617

**Comment by kavithagurusamy** [comment]
Fixed in OrangeHRM 4.3.5 version
